errors/handlers.js

```diff
@@ -0,0 +1,9 @@
+exports.handle500 = (err, req, res, next) => {
+    console.log(err)
+    res.status(500).send("apologies for my wonky code")
```

Comment:
We should send the error message back on a key of msg to be consistent

Reply:
Have amended, good to merge?